pom.xml

```diff
@@ -189,5 +189,38 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>loopback</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <version>2.5.5</version>
+                        <configuration>
+                            <finalName>loopback-proxy</finalName>
+                            <archive>
+                                <manifest>
+                                    <mainClass>com.github.monkeywie.proxyee.LoopbackProxyServerApp</mainClass>
+                                </manifest>
+                            </archive>
+                            <descriptorRefs>
+                                <descriptorRef>jar-with-dependencies</descriptorRef>
+                            </descriptorRefs>
+                            <appendAssemblyId>false</appendAssemblyId>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>make-assembly</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>single</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
This is a library project, these should be done by the user who used this library.

Reply:
I agree.